dataplane/ebpf/src/egress/icmp.rs

```diff
@@ -38,16 +38,8 @@ pub fn handle_icmp_egress(ctx: TcContext) -> Result<i32, i64> {
     
     let new_src = unsafe { BLIXT_CONNTRACK.get(&dest_addr) }.ok_or(TC_ACT_PIPE)?;
 
-    let daddr_dot_dec = ip_from_int(unsafe { (*ip_hdr).daddr });
-    info!(
-        &ctx,
-        "Received a ICMP Unreachable packet destined for svc ip: {}.{}.{}.{}",
-        daddr_dot_dec[0],
-        daddr_dot_dec[1],
-        daddr_dot_dec[2],
-        daddr_dot_dec[3],
-    );
-
+    info!(&ctx, "Received a ICMP Unreachable packet destined for svc ip:: {:ipv4} ", u32::from_be(unsafe { (*ip_hdr).daddr }));
```

Comment:
```suggestion
    info!(&ctx, "Received a ICMP Unreachable packet destined for svc ip: {:ipv4} ", u32::from_be(unsafe { (*ip_hdr).daddr }));
```

Reply:
Damn sorry for this typo u had to fix shane. Thanks accepted it.